example/csv/src/main/java/org/apache/calcite/adapter/csv/CsvFilterableTable.java

```diff
@@ -63,7 +63,11 @@ public String toString() {
   }
 
   private boolean addFilter(RexNode filter, Object[] filterValues) {
-    if (filter.isA(SqlKind.EQUALS)) {
+    if (filter.isA(SqlKind.AND)) {
```

Comment:
What about the predicate `OR`?

Reply:
I think `OR` is useless for generating `filterValues`. When o.a.c.a.c.CsvEnumerator#moveNext read rows from source, it will filter the rows which isn't match `filterValues`.
If we do, the rows are suited for others operands of `OR` will be filtered by `filterValues`.